Makefile

```
@@ -419,11 +423,15 @@ $(CLEAN_TARGETS):
-clean_all: $(CLEAN_TARGETS)
+## clean_all         : clean all targets (compiled objects only; obj/*.hex and obj/*.bin preserved)
+clean_all:
+	@echo "Cleaning all targets"
+	$(V0) rm -rf $(OBJECT_DIR)
+	$(V0) cd src/test && $(MAKE) clean || true
+	@echo "All targets cleaned."
 
-## all_clean         : clean all valid targets (alias for above)
-all_clean: $(TARGETS_CLEAN)
+## all_clean         : clean all targets (alias for clean_all)
+all_clean: clean_all
```

checkmake (0.3.2)

[warning] Target "all_clean" should be declared PHONY.

(phonydeclared)